components/eam/cime_config/config_component.xml

```diff
@@ -53,6 +53,7 @@
       <value compset="_EAM"                    >-phys default</value>
       <value compset="_EAM%CMIP6-1pctCO2"      >&eamv3_phys_defaults; &eamv3_chem_defaults;</value>
       <value compset="_EAM%CMIP6-4xCO2"        >&eamv3_phys_defaults; &eamv3_chem_defaults;</value>
+      <value compset="20TR.*_EAM%CMIP6-.*_"    >&eamv3_phys_defaults; &eamv3_chem_defaults;</value>
```

Comment:
This is causing a dashboard failure for MMF cases because the coupled MMF cases uses "CMIP6-MMF" - so we need to change this.